specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-03-01-preview/nova.json

```diff
@@ -0,0 +1,1058 @@
+          },
+          {
+            "$ref": "#/parameters/dataCenterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DataCenterResource"
+            },
+            "description": "Parameters to provide for specifying the managed Cassandra data center."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. Update will proceed asynchronously."
+          },
+          "200": {
+            "description": "Completed synchronously. This will only happen if the fields changed by patch require no changes to the actual Cassandra cluster, such as editing the tags."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "clusterNameParameter": {
+      "name": "clusterName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "description": "Managed Cassandra cluster name.",
+      "minLength": 1,
+      "maxLength": 100,
+      "pattern": "^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*"
+    },
+    "dataCenterNameParameter": {
+      "name": "dataCenterName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "description": "Data center name in a managed Cassandra cluster.",
+      "minLength": 1,
+      "maxLength": 100,
+      "pattern": "^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*"
+    },
+    "backupNameParameter": {
+      "name": "backupId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "description": "Id of a restorable backup of a Cassandra cluster.",
+      "minLength": 1,
+      "maxLength": 15,
+      "pattern": "^[0-9]+$"
+    }
+  },
+  "definitions": {
+    "ListClusters": {
+      "description": "List of managed Cassandra clusters.",
+      "readOnly": true,
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Container for the array of clusters.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClusterResource"
+          }
+        }
+      }
+    },
+    "ManagedCassandraProvisioningState": {
+      "description": "The status of the resource at the time the operation was called.",
+      "type": "string",
+      "enum": [
+        "Creating",
+        "Updating",
+        "Deleting",
+        "Succeeded",
+        "Failed",
+        "Cancelled"
```

Comment:
This should be Canceled with one el (see https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#provisioningstate-property)